### PR TITLE
Feat/rename table

### PR DESCRIPTION
**Pull request by djklim87**

### CODE REVIEW

**Review by donhardman — CHANGES_REQUESTED** [review]
We should move this plugin into the `buddy-plugin-modify-table`, because that plugin is designed to handle table modifications (including alters).

File: src/Plugin/RenameTable/Payload.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+		 *                           delim: bool,
+		 *                           parts: array<string>
+		 *                       },
+		 *                       base_expr: string
+		 *                   }
+		 *               }>
+		 *           }
+		 *       } $payload
+		 */
+		$payload = Payload::$sqlQueryParser::parse($request->payload);
```
[the remark this answers is not in the document]
Comment (djklim87) [reply src/Plugin/RenameTable/Payload.php]: done

**Comment by djklim87** [comment]
> We should move this plugin into the `buddy-plugin-modify-table`, because that plugin is designed to handle table modifications (including alters).

As discussed at last dev call we decide to separate plugins to keep it simple and clear

**Review by donhardman — APPROVED** [review]
Looks good for me

**Pull request merged**